src/version.cpp

```diff
@@ -37,38 +37,38 @@ const std::string CLIENT_NAME("Satoshi");
 
 // First, include build.h if requested
 #ifdef HAVE_BUILD_INFO
-#    include "build.h"
+#include "build.h"
```

Comment:
I don't mind these. I'd have liked to see all of it indented though. 

```c
#ifdef HAVE_BUILD_INFO
    #include "build.h"
```

That would make me happy if it's possible.